tests/pds/ingress/util/test_path_util.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import os
+import tempfile
+import unittest
+from os.path import abspath
+from os.path import join
+
+from pkg_resources import resource_filename
+
+from src.pds.ingress.util.path_util import PathUtil
+
+
+class PathUtilTest(unittest.TestCase):
```

Comment:
Good that you develop unit test. we would like to have metrics on code coverage in the future.